.github/workflows/custom_docker_builds.yml

```diff
@@ -46,21 +46,25 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v3
 
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v2
+
       - name: Log in to the Container registry
-        uses: docker/login-action@d398f07826957cd0a18ea1b059cf1207835e60bc
+        uses: docker/login-action@v2
```

Comment:
Should we stick with the practice of using the sha here, as github [recommends](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-third-party-actions)?  

We could, however, include a line-end comment with the corresponding version.  I think dependabot can handle that, including updating the version comment, so if we're (justifiably) worried about falling behind, we might lean on dependabot for that.

I'm not insisting, maybe just curious what your thoughts are.

Reply:
Ah i didn't know github recommends that. In that case, I agree we should do that - I've updated the PR.

Reply:
Hmm I did a `git push`, but I'm not seeing it in the github ui...

Reply:
Not sure whats going with github, but you can see the new changes if you view my branch directly https://github.com/spack/spack-infrastructure/tree/build-arm-images